components/Blocks/PhotoHero.css

```diff
@@ -0,0 +1,46 @@
+.root {
+  width: 100%;
+  padding: 5em 0;
+  position: relative;
+  background-size: cover;
+  color: var(--color-white);
```

Comment:
I'm not too fussed at this point. If we drastically change the PhotoHero itself (changing the bg gradient, for example) then we'll have to change up a load of other CSS in the same file anyway. Extrapolating the variable out to another variable doesn't really solve an issue.